src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,313 @@
+    url: /automate-workflows/5-mins-tag-resources
+  - title: 'Tagging: Use tags to organize and group what you monitor'
+    url: https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/tagging-use-tags-organize-group-what-you-monitor
+---
+
+At New Relic, we are always looking for challenging problems to solve. One of the things we've been working on lately is figuring out ways for our developers to experience different aspects of New Relic so they can more quickly build on the platform, find a solution to a specific use case, or just learn something new (which is always fun!). 
+
+One of the frameworks that we are working on enables our various teams to define and spin up complete environments, from infrastructure to applications, to instrumentation. Like many internal frameworks, this has undergone several iterations, and in fact, we are in the midst right now of major refactor. Because of our commitment to open source, we wanted to start working openly early in the process and invite you to collaborate with us. Drop any thoughts & comments in the [Build on New Relic](https://discuss.newrelic.com/c/build-on-new-relic) and let us know what you think! 
+
+OK, let's jump in!
+
+## Overview
+
+One of the things that we've learned is that keeping your stack organized can be challenging, especially as resources come and go. One strategy to help with this is to apply tags in a consistent manner across your estate. In this guide, you will provision a host on your cloud provider, deploy a simple node.js app on that host, instrument both with New Relic agents, and then apply a consistent tagging strategy across all the resources--all 'automagically'. (Not really, it's mostly Ansible under the covers.) 
+
+**Remember** Make sure to do the **Teardown** step at the end to decommision the entities that we provision in this guide. 
+
+
+## Before you begin
+To run this example there are several pre-requisite steps, most of which involve setting up the right configurations, and at least for now, building a [docker](https://docs.docker.com/get-docker/) image. 
+
+We use several of our open source projects to make this all work. Let's take a look at them now.
+* [Demo Deployer](https://github.com/newrelic/demo-deployer) - This is the engine that drives all the automation. For convenience, you'll clone this repo and build a docker image. In future iterations, we'll publish the image in docker hub or someplace similar. 
+* [Demo Nodetron] (https://github.com/newrelic/demo-nodetron) - This is the simple app that will be provisioned on a host. It will give you a simple web page and an API that you can hit with a tool like [Postman](https://www.postman.com/downloads/) depending on how much traffic you want to generate. 
+* [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation) - This is the project that we use to layer in the instrumentation for the infrastructure and the application. 
```

Comment:
Should New and Relic be attached (no space)?

Reply:
I don't think so b/c it's a label.